### PR TITLE
Cap OpenSSL_jll to 1.1.10 for old JLLs

### PR DESCRIPTION
**Pull request by mkitti**
Continuation of #78542

Cap compat of older JLLs at OpenSSL_jll 1.1.10 until they can be rebuilt for OpenSSLv3

### CODE REVIEW

**Comment by mkitti** [comment]
@giordano does this look right to you?

File: jll/L/librdkafka_jll/Compat.toml [hunk 1]
```diff
@@ -1,12 +1,8 @@
-[1]
+[1-2]
 OpenSSL_jll = "1.1.10-1"
 
 ["1-1.5"]
 julia = "1"
 
 ["1.8-2"]
-JLLWrappers = "1.2.0-1"
 julia = "1.6.0-1"
```
Comment (giordano) [inline jll/L/librdkafka_jll/Compat.toml:8]: Why this is gone?

Comment (giordano) [reply jll/L/librdkafka_jll/Compat.toml:8]: Well, JLLWrappers, commenting on the phone is hard

**Review by mkitti — COMMENTED** [review]
Restore JLLWrappers

File: jll/L/librdkafka_jll/Compat.toml (hunk 1 above)
Comment (mkitti) [reply jll/L/librdkafka_jll/Compat.toml:9]: Oops, I put it back

**Pull request merged**